src/test/java/uk/ac/ebi/ampt2d/registry/PropertyRegistryServiceApplicationTests.java

```diff
@@ -126,7 +137,7 @@ public void shouldQueryPhenotype() throws Exception {
     public void shouldUpdatePhenotype() throws Exception {
         String location = postTestPhenotype();
 
-        mockMvc.perform(put(location).with(oAuthHelper.bearerToken("testEditor@gmail.com"))
+        mockMvc.perform(patch(location).with(oAuthHelper.bearerToken("testEditor@gmail.com"))
```

Comment:
Should there be any test to update newly introduced fields

Reply:
done in 9b6b41d